lib/rubygems/util/licenses.rb

```diff
@@ -8,416 +8,420 @@ class Gem::Licenses
-      JasPer-2.0
-      LAL-1.2
-      LAL-1.3
-      LGPL-2.0
-      LGPL-2.0+
-      LGPL-2.0-only
-      LGPL-2.0-or-later
-      LGPL-2.1
-      LGPL-2.1+
-      LGPL-2.1-only
-      LGPL-2.1-or-later
-      LGPL-3.0
-      LGPL-3.0+
-      LGPL-3.0-only
-      LGPL-3.0-or-later
-      LGPLLR
-      LPL-1.0
-      LPL-1.02
-      LPPL-1.0
-      LPPL-1.1
-      LPPL-1.2
-      LPPL-1.3a
-      LPPL-1.3c
-      Latex2e
-      Leptonica
-      LiLiQ-P-1.1
-      LiLiQ-R-1.1
-      LiLiQ-Rplus-1.1
-      Libpng
-      MIT
-      MIT-CMU
-      MIT-advertising
-      MIT-enna
-      MIT-feh
-      MITNFA
-      MPL-1.0
-      MPL-1.1
-      MPL-2.0
-      MPL-2.0-no-copyleft-exception
-      MS-PL
-      MS-RL
-      MTLL
-      MakeIndex
-      MirOS
-      Motosoto
-      Multics
-      Mup
-      NASA-1.3
-      NBPL-1.0
-      NCSA
-      NGPL
-      NLOD-1.0
-      NLPL
-      NOSL
-      NPL-1.0
-      NPL-1.1
-      NPOSL-3.0
-      NRL
-      NTP
-      Naumen
-      Net-SNMP
-      NetCDF
-      Newsletr
-      Nokia
-      Noweb
-      Nunit
-      OCCT-PL
-      OCLC-2.0
-      ODbL-1.0
-      OFL-1.0
-      OFL-1.1
-      OGTSL
-      OLDAP-1.1
-      OLDAP-1.2
-      OLDAP-1.3
-      OLDAP-1.4
-      OLDAP-2.0
-      OLDAP-2.0.1
-      OLDAP-2.1
-      OLDAP-2.2
-      OLDAP-2.2.1
-      OLDAP-2.2.2
-      OLDAP-2.3
-      OLDAP-2.4
-      OLDAP-2.5
-      OLDAP-2.6
-      OLDAP-2.7
-      OLDAP-2.8
-      OML
-      OPL-1.0
-      OSET-PL-2.1
-      OSL-1.0
-      OSL-1.1
-      OSL-2.0
-      OSL-2.1
-      OSL-3.0
-      OpenSSL
-      PDDL-1.0
-      PHP-3.0
-      PHP-3.01
-      Plexus
-      PostgreSQL
-      Python-2.0
-      QPL-1.0
-      Qhull
-      RHeCos-1.1
-      RPL-1.1
-      RPL-1.5
-      RPSL-1.0
-      RSA-MD
-      RSCPL
-      Rdisc
-      Ruby
-      SAX-PD
-      SCEA
-      SGI-B-1.0
-      SGI-B-1.1
-      SGI-B-2.0
-      SISSL
-      SISSL-1.2
-      SMLNJ
-      SMPPL
-      SNIA
-      SPL-1.0
-      SWL
-      Saxpath
-      Sendmail
-      SimPL-2.0
-      Sleepycat
-      Spencer-86
-      Spencer-94
-      Spencer-99
-      StandardML-NJ
-      SugarCRM-1.1.3
-      TCL
-      TCP-wrappers
-      TMate
-      TORQUE-1.1
-      TOSL
-      UPL-1.0
-      Unicode-DFS-2015
-      Unicode-DFS-2016
-      Unicode-TOU
-      Unlicense
-      VOSTROM
-      VSL-1.0
-      Vim
-      W3C
-      W3C-19980720
-      W3C-20150513
-      WTFPL
-      Watcom-1.0
-      Wsuipa
-      X11
-      XFree86-1.1
-      XSkat
-      Xerox
-      Xnet
-      YPL-1.0
-      YPL-1.1
-      ZPL-1.1
-      ZPL-2.0
-      ZPL-2.1
-      Zed
-      Zend-2.0
-      Zimbra-1.3
-      Zimbra-1.4
-      Zlib
-      bzip2-1.0.5
-      bzip2-1.0.6
-      curl
-      diffmark
-      dvipdfm
-      eCos-2.0
-      eGenix
-      gSOAP-1.3b
-      gnuplot
-      iMatix
-      libtiff
-      mpich2
-      psfrag
-      psutils
-      wxWindows
-      xinetd
-      xpp
-      zlib-acknowledgement
-  ).freeze
+  LICENSE_IDENTIFIERS = {
```

Comment:
please add documentation for what the values in this hash mean

Reply:
Thank you for reviewing! Added documentation in ca3ddd2.